app/controllers/cmsimple/paths_controller.rb

```diff
@@ -12,7 +12,9 @@ def index
 
     def create
       @path = Path.new(path_params)
+      require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.